docs/UserGuide.adoc

```diff
@@ -247,26 +247,53 @@ Format: `redo`
 * Allows redoing up until the most recent change.
 
 //<!-- @@author A0132788U -->
-=== Set storage path : `set`
+=== Save storage path : `save`
 
-Sets to user entered filepath. +
-Format: `set ./filepath.xml`
+Saves entrybook data to user entered filepath. +
+Format: `save ./filepath.xml`
```

Comment:
maybe it'll be better to write as save ./filepath/filename.xml for better clarity